### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by francescorubbo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,49 @@
 Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
```
Comment (adi-prakash-ml) [inline README.md]: Maybe leave out "phenotypic". I have always seen this referred to as HCS or HCI. Phenotypic is more of the intent that such features will come out after the experiment.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,49 @@
...
 This dataset is designed to benchmark the compound similarity method by allowing measurements of inherent sources of variability such as donors, plates, and experimental runs. We used a bespoke library of 623 compounds, profiled in 4 batches of 8 plates each across a total of 16 donors.
+
+## Explore the data
+
+The data can be visualized and analyzed in the Spring platform at https://app.springscience.com/explore-iinflammasome-benchmark.
+
+## Download the data
+
+The raw image data, along with field metadata and pre-computed features can be accessed at [link coming soon..,].
+The dataset is organized in directories, one for each of the four batches of experiments.
```
Comment (adi-prakash-ml) [inline README.md:17]: Mention that the they are named as dates for when the experiment was performed.

File: README.md [hunk 3]
```diff
@@ -3,3 +3,49 @@
...
+Each directory includes the following items:
+- The `raw_images` directory contains 2048x2048 16bit TIFF files organized in directories for each plate. The file names encode the location in the plate (`<well>_<field>`, e.g. `A01_f00`), as well as the timepoint `t`, the z-layer `z` and the channel `c` (e.g. `t0z0c0`). Note that for this dataset all images are taken at the same timepoint (`t0`) and z-layer (`z0`).
```
Comment (adi-prakash-ml) [inline README.md:19]: Should we tell them what plate and imager were used?

File: README.md [hunk 4]
```diff
@@ -3,3 +3,49 @@
 Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
 
 This dataset is designed to benchmark the compound similarity method by allowing measurements of inherent sources of variability such as donors, plates, and experimental runs. We used a bespoke library of 623 compounds, profiled in 4 batches of 8 plates each across a total of 16 donors.
+
+## Explore the data
+
+The data can be visualized and analyzed in the Spring platform at https://app.springscience.com/explore-iinflammasome-benchmark.
+
+## Download the data
```
Comment (adi-prakash-ml) [inline README.md:13]: With every dataset or data download it is important to provide the hash values to make sure users got the right data and it has not been tampered with or incomplete. GCS will give you the MD5 and CRC32 if you see the details tab.

File: README.md [hunk 5]
```diff
@@ -3,3 +3,49 @@
...
+
+The raw image data, along with field metadata and pre-computed features can be accessed at [link coming soon..,].
+The dataset is organized in directories, one for each of the four batches of experiments.
+Each directory includes the following items:
+- The `raw_images` directory contains 2048x2048 16bit TIFF files organized in directories for each plate. The file names encode the location in the plate (`<well>_<field>`, e.g. `A01_f00`), as well as the timepoint `t`, the z-layer `z` and the channel `c` (e.g. `t0z0c0`). Note that for this dataset all images are taken at the same timepoint (`t0`) and z-layer (`z0`).
+- The `immunofluorescence_metadata.json` file provides the information about the immunofluorescence dyes used in the experiment. The order in the `palette` arrays corresponds to `channel` numeral in the image names.
+- The `well_metadata.parquet` file contains the metadata information for each well in the experiment. See below for details about the metadata fields.
+- The `features` directory contains `.parquet` files with pre-computed features for each cell. The files are organized in directories for each plate.
+
+## Metadata
```
Comment (adi-prakash-ml) [inline README.md:24]: Only optional, so happy to push back. Do you think we should provide all the unique values for each field? Maybe or maybe not, but it seems like a good high-level view of the data without downloading. Very few people will download such large data and will need to know more about it before committing to the download.

**Review by adi-prakash-ml — APPROVED** [review]
I left a few comments - nothing too critical, so :okay: to publish from my side.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,54 @@
...
+
+More details about the dataset can be found in our [SLAS2023 poster](https://www.springdiscovery.com/pdf/SLAS23-Poster-2-Inflammasome-Benchmark.pdf).
+
+## Explore the data
+
+The data can be visualized and analyzed on the [Spring platform](https://app.springscience.com/explore-iinflammasome-benchmark).
+
+## Download the data
+
+The raw image data, along with field metadata and pre-computed features can be accessed from the public [spring-inflammasom-benchmark-dataset](https://console.cloud.google.com/storage/browser/spring-inflammasome-benchmark-dataset) GCS bucket.
```
Comment (benkomalo) [inline README.md:15]: typo here

File: README.md [hunk 7]
```diff
@@ -1,5 +1,54 @@
 # Spring Inflammasome Benchmark Dataset
 
-Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
+Spring conducted a high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
```
Comment (benkomalo) [inline README.md:3]: maybe: "...Using images from these datasets, we established computational methods to score phenotypic similarity between compounds in order to identify mechanistic similarities between them."

("biologically related" is a bit vague...)

File: README.md [hunk 8]
```diff
@@ -1,5 +1,54 @@
...
+
+The data can be visualized and analyzed on the [Spring platform](https://app.springscience.com/explore-iinflammasome-benchmark).
+
+## Download the data
+
+The raw image data, along with field metadata and pre-computed features can be accessed from the public [spring-inflammasom-benchmark-dataset](https://console.cloud.google.com/storage/browser/spring-inflammasome-benchmark-dataset) GCS bucket.
+
+The dataset is organized in directories, one for each of the four batches of experiments.
+Each directory is named with the date the experiment was executed, and includes the following items:
+- The `raw_images` directory contains 2048x2048 16bit TIFF files organized in directories for each plate. The file names encode the location in the plate (`<well>_<field>`, e.g. `A01_f00`), as well as the timepoint `t`, the z-layer `z` and the channel `c` (e.g. `t0z0c0`). Note that for this dataset all images are taken at the same timepoint (`t0`) and z-layer (`z0`).
```
Comment (benkomalo) [inline README.md:19]: should also maybe specify the timepoint.
I'm also not sure if "all images are taken at the same z-layer" is also an accurate statement. We did do individual z-slices and do a max projection right? So it's probably worth clarifying that we did that.

File: README.md [hunk 9]
```diff
@@ -1,5 +1,54 @@
...
+- The `immunofluorescence_metadata.json` file provides the information about the immunofluorescence dyes used in the experiment. The order in the `palette` arrays corresponds to the `channel` numeral in the image names.
+- The `well_metadata.parquet` file contains the metadata information for each well in the experiment. See below for details about the metadata fields.
+- The `features` directory contains `.parquet` files with pre-computed features for each cell. The files are organized in directories for each plate.
```
Comment (benkomalo) [inline README.md:22]: One thing that's weird is: our single-cell indexing is reversed, such that row=x and column=y positions. :( 

We should either clean that up (i.e. rename the actual indices) or at least document

**Review by benkomalo — COMMENTED** [review]
Sorry for the delay!

**Pull request merged**